### PR TITLE
added bms and car state

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: client/app/dashboard/overview/overview.component.js [hunk 1]
```diff
@@ -25,6 +25,16 @@ function updateTemperatures($scope,temp) {
 
 }
 
+function updateStates(bms,car)
+{
+  if(bms){
+    angular.element(document.querySelector('#bms-state')).html("BMS State: " + bms.flag);
+  }
+  if(car) {
+    angular.element(document.querySelector('#car-state')).html("Car State: " + car.state);
```
Comment (klanmiko) [inline client/app/dashboard/overview/overview.component.js]: car.state is an integer

Can you convert from the integer to a string

File: client/app/dashboard/overview/overview.component.js [hunk 2]
```diff
@@ -25,6 +25,16 @@ function updateTemperatures($scope,temp) {
 
 }
 
+function updateStates(bms,car)
+{
+  if(bms){
+    angular.element(document.querySelector('#bms-state')).html("BMS State: " + bms.flag);
```
Comment (klanmiko) [inline client/app/dashboard/overview/overview.component.js]: bms.flag is an array of true and false values

Need you to convert this to a string also. The CAN doc contains it

**Review by klanmiko — CHANGES_REQUESTED** [review]
need to convert from raw data values to strings

**Comment by DanG100** [comment]
made changes

**Review by klanmiko — APPROVED** [review]

**Pull request merged**